.pullapprove.yml

```diff
@@ -1,10 +1,14 @@
-approve_by_comment: true
-approve_regex: '(Approved|:shipit:|:\+1:|LGTM)'
-author_approval: ignored
-reject_regex: ^Rejected
-reset_on_push: true
-reviewers:
-  name: maintainers
-  required: 2
-  teams:
-    - maintainers
+version: 2
+groups:
+  maintainers:
+    required: 3
+    author_approval:
+      auto: true
+    approve_by_comment: 
+      enabled: true
+      approve_regex: '^(Approved|:shipit:|:\+1:|LGTM)'
```

Comment:
Do we want to require the approval at the beginning on the comment ( ^ ), didn't have it before